src/main/resources/META-INF/plugin.xml

```diff
@@ -23,12 +23,20 @@
     <depends>org.jetbrains.kotlin</depends>
 
     <extensions defaultExtensionNs="com.intellij">
-        <localInspection implementationClass="com.github.suusan2go.kotlinfillclass.inspections.FillClassInspection"
+        <localInspection implementationClass="com.github.suusan2go.kotlinfillclass.inspections.FillDummyValuesInspection"
```

Comment:
@theapache64 if i may make a tiny ergonomics request, would be possible to keep the old display name and extend when needed? as well as keeping the current behavior as first option? when i code i run entirely on visual pattern recognition :o)

ie:
```
Fill constructor
Fill constructor with dummy values

Fill function
Fill function with dummy values
```

Reply:
@sphrak Sure. Does [2a791a2](https://github.com/suusan2go/kotlin-fill-class/pull/84/commits/2a791a2f2903a47c60e7705e0ba9c421c9d09129) looks good to you?

Reply:
@theapache64 yes lgtm, thank you. this is gonna be so nice to have when writing tests and stuff